utils/clusterutils.go

```diff
@@ -99,9 +102,19 @@ func getDaemonset(deployment *appsv1.Deployment) *appsv1.DaemonSet {
 				Spec: corev1.PodSpec{
 					NodeSelector:                  cfg.NodeSelector,
 					TerminationGracePeriodSeconds: &terminationGracePeriodSeconds,
-					Containers:                    getContainers(),
-					ImagePullSecrets:              imgPullSecrets,
-					Affinity:                      cfg.Affinity,
+					InitContainers: []corev1.Container{{
+						Name:            "copy-sleep",
+						Image:           "quay.io/eclipse/kubernetes-image-puller:next",
```

Comment:
Do we need to use something like https://github.com/che-incubator/kubernetes-image-puller/blob/fa84e441c0b1e050c2c4957eccb655f0d2d5c07f/cfg/envvars.go#L137 for the image?

Reply:
yes, probably a good idea and extract to `defaultImage` const

Reply:
thank you, updated in latest commit